github-orgs/kubeflow/org.yaml

```diff
@@ -628,6 +628,12 @@ orgs:
             - tmckayus
             - pdmack
             privacy: closed
+          SUSE:
+            description: Team at SUSE contributing to Kubeflow.
+            members:
+            - alfsuse
```

Comment:
You'll need to add all members of the team to the organization as members.

Reply:
@jlewi Added all members to the list in the last commit, we may eventually need to update the list in the future but I'll re-open another PR if that is the case.